src/ontology/Makefile

```diff
@@ -159,7 +161,14 @@ fma_uberon.owl: fma_import_manual.txt
 uberon_human_extended.owl: fma_uberon.owl uberon_human.owl
 	$(ROBOT) merge --input fma_uberon.owl --input uberon_human.owl relax reduce --output $@
 
+uberon_axioms_in_cl.ofn: mirror/cl.owl
+	$(ROBOT) merge -i $< remove --base-iri http://purl.obolibrary.org/obo/UBERON_ --axioms external --trim false -o $@
 
+uberon_axioms_in_cl.owl: uberon_axioms_in_cl.ofn
+	sed '/^Declaration/d' $< > $@
+
+cl_import.owl: mirror/cl.owl uberon_axioms_in_cl.owl
+	$(ROBOT) merge -i $< unmerge -i uberon_axioms_in_cl.owl -o $@
 
```

Comment:
If I understand correctly cl_import.owl now = all of CL but minus uberon_axioms in CL.  If so, this looks good.